sources/packages/api/src/route-controllers/student-account-applications/student-account-application.aest.controller.ts

```diff
@@ -160,10 +160,12 @@ export class StudentAccountApplicationAESTController extends BaseController {
   async declineStudentAccountApplication(
     @Param("studentAccountApplicationId", ParseIntPipe)
     studentAccountApplicationId: number,
+    @UserToken() userToken: IUserToken,
```

Comment:
Looks like @param description on the userToken is missing.

Reply:
i think we don't add comment for `userToken`

Reply:
As mentioned in previous PRs reviews and as agreed we would not be adding those.